homeassistant/components/squeezebox/strings.json

```
@@ -103,5 +103,16 @@
         "unit_of_measurement": "[%key:component::squeezebox::entity::sensor::player_count::unit_of_measurement%]"
       }
     }
+  },
+  "options": {
+    "step": {
+      "init": {
+        "title": "LMS Configuration.",
+        "data": {
+          "browse_limit": "Number of items to return from media browser.",
```

This option is being used to drive 2 different things in the implementation:

```suggestion
          "browse_limit": "Maximum number of items to include when browsing media and generating a playlist.",
```

---

Thanks